README.md

```diff
@@ -31,82 +69,134 @@ Add this plugin like this:
-  console.log(metafeed) // { seed, keys }
+const find = (feed) => (
+  feed.feedpurpose === 'aboutMe' &&
+  feed.feedformat === 'classic'
+)
+const details = {
+  feedpurpose: 'aboutMe',
+  feedformat: 'classic',
+}
+
+sbot.metafeeds.findOrCreate(null, find, details, (err, aboutMeFeed) => {
+  console.log(rootFeed)
+
+  // publish things to aboutMeFeed!
 })
 ```
 
-Now this has created the `seed`, which in turn is used to generate an [ssb-keys]
-`keys` object. The `seed` is actually also published on the _main_ feed as a
-private message to yourself, to allow recovering it in the future.
+Once you have these the *FeedDetail* object `aboutMeFeed` you can publish to that feed:
 
-There can only be one _root_ meta feed, so even if you call `findOrCreate` many
-times, it will not create duplicates, it will just load the meta feed
-`{ seed, keys }`.
+```js
+const content = {
+  type: 'about',
+  about: rootFeed.keys.id,
+  name: 'baba yaga'
+  description: 'lives in a hutt in the forest, swing by sometime!'
+}
+sbot.db.publishAs(aboutMeFeed.keys, content, cb)
+```
+
+
+## API
+
+### `sbot.metafeeds.findOrCreate(cb)`
 
-Now you can create subfeeds _under_ that root meta feed by passing three
-arguments to `findOrCreate`, before the callback, like this:
+Calls back with your `root` Metafeed object which has the form:
 
 ```js
-const details = {
-  feedpurpose: 'mygame',
-  feedformat: 'classic',
-  metadata: {
-    score: 0,
-    whateverElse: true,
+{
+  metafeed: null,
+  feedpurpose: 'root',
+  feedtype: 'bendybutt-v1',
+  seed: <Buffer 13 10 25 ab e3 37 20 57 19 0a 1d e4 64 13 e7 38 d2 23 11 48 7d 13 e6 3b 8f ef 72 92 7f db 96 64>,
+  id: 'ssb:feed/bendybutt-v1/sxK3OnHxdo7yGZ-28HrgpVq8nRBFaOCEGjRE4nB7CO8=',
+  keys: {
+    curve: 'ed25519',
+    public: 'sxK3OnHxdo7yGZ+28HrgpVq8nRBFaOCEGjRE4nB7CO8=.ed25519',
+    private: 'SOEx7hA9vRHrli0PZwNJ8jijH+PShmlrzz/JAKI7v6SzErc6cfF2jvIZn7bweuClWrydEEVo4IQaNETicHsI7w==.ed25519',
+    id: 'ssb:feed/bendybutt-v1/sxK3OnHxdo7yGZ-28HrgpVq8nRBFaOCEGjRE4nB7CO8='
   },
-}
-
-const visit = f => f.feedpurpose === 'mygame' && f.feedformat === 'classic'
-
-sbot.metafeeds.findOrCreate(metafeed, visit, details, (err, subfeed) => {
-  console.log(subfeed)
-  // {
-  //   feedpurpose: 'mygame',
-  //   subfeed: '@___________________________________________=.ed25519',
-  //   keys: {
-  //     curve,
-  //     public,
-  //     private,
-  //     id
-  //   }
-  // }
-})
+  metadata: {}
 ```
 
-This has created a `keys` object for a new subfeed, which you can use to publish
-application-specific messages (such as for a game). The `details` argument
-always needs `feedpurpose` and `feedformat` (supports `classic` for ed25519
-normal SSB feeds, and `bendybutt-v1`).
+Meaning:
+- `metafeed` - the id of the feed this is underneath. As this is the topmost feed, this is empty
+- `feedpurpose` - a human readable ideally unique handle for this feed
+- `feedtype` - the feed format ("classic" or "bendybutt-v1" are current options)
```

Comment:
Where did you find `feedtype`? I grepped ssb-meta-feeds and couldn't find it.

Reply:
Hmmm, not sure.

Reply:
Yeah this is weird. I remember thinking "this shouldn't be type, it's more like bfe format..."
Then I wrote the wrong one?
Strange